slack-app-backend/src/main/java/com/slack/api/app_backend/events/servlet/SlackEventsApiServlet.java

```diff
@@ -17,6 +17,7 @@
 import java.util.Locale;
 import java.util.stream.Collectors;
 
+@Deprecated // Please consider Bolt framework with bolt-servlet adapter instead
```

Comment:
This module is a legacy one, which was added before bolt-java development. I don't think that many people are using this but now it's a good timing to mark this simple module as deprecated and encourage people to use bolt-java for new apps.

I'm not planning to remove this class because we're not in a hurry to remove it as the javax.servlet dependency in this module is optional (=provided scope).